spec/author_spec.rb

```diff
@@ -0,0 +1,34 @@
+require_relative '../models/author'
+
+describe Author do
+  before :each do
+    @author = Author.new('David', 'Karp')
+  end
+
+  it 'Should be instance of "Author"' do
+    expect(@author).to be_instance_of Author
+  end
+
+  it 'first_name should return "David"' do
+    expect(@author.first_name).to eq 'David'
+  end
+
+  it 'last_name should return "Karp"' do
+    expect(@author.last_name).to eq 'Karp'
+  end
+
+  it 'items count should be 0' do
+    expect(@author.items.count).to eq 0
+  end
+
+  describe '#add_item' do
+    it 'items count should become 1' do
+      book = double('book')
+      allow(book).to receive(:add_author).with(@author).and_return(true)
```

Comment:
Hi @ksigei 👋 

Please try to follow a good naming convention. You are working `Game` and `Author` so the name of the **mock** should be **game** and not **book**

Reply:
@RitobrotoMukherjee thanks for correction, done